### PR TITLE
Update the incomplete steps to add a custom header column

### PR DESCRIPTION
**Pull request by pankajparashar**
Source:
https://devtoolstips.org/tips/en/custom-headers-in-network-table/

AB#45387340

### CODE REVIEW

**Comment by captainbrosset** [comment]
Thank you @pankajparashar for submitting this PR. I'll make sure we review and merge this soon.

File: microsoft-edge/devtools-guide-chromium/network/reference.md [hunk 1]
```diff
@@ -350,7 +350,7 @@ Right-click the header of the Requests table and select an option to hide or sho
 
 #### Add custom columns
 
-To add a custom column to the Requests table, right-click the header of the Requests table and then select **Response Headers** > **Manage Header Columns**.
+To add a custom column to the Requests table, right-click the header of the Requests table and then select **Response Headers** > **Manage Header Columns**. In the **Manage Header Columns** popup window, click **Add Custom Header**. Enter the custom header name and click **Add** to add a new column to the requests table.
```
Comment (mikehoffms) [inline microsoft-edge/devtools-guide-chromium/network/reference.md:353]:
```suggestion
To add a custom column to the Requests table, right-click the header of the Requests table and then select **Response Headers** > **Manage Header Columns**. The **Manage Header Columns** popup window opens.  Click the **Add custom header** button, enter the custom header name, and then click **Add**. 
```
brought wording closer to standard action > result UI format.  Lowercased button name per UI label.  

gh rendered section (commit-specific):
https://github.com/MicrosoftDocs/edge-developer/blob/447e4f1c612cee93f1fc5f3cab2a8785cf86098c/microsoft-edge/devtools-guide-chromium/network/reference.md#add-custom-columns

**Review by mikehoffms — APPROVED** [review]
Writer/Editor pass

**Comment by mikehoffms** [comment]
offline, i requested merging this PR

**Review by captainbrosset — APPROVED** [review]

**Comment by captainbrosset** [comment]
All merged and published to the live branch. Should appear on the live site any time now. Thank you!

**Pull request merged**